### PR TITLE
update duckduckgo.po

### PR DESCRIPTION
**Pull request by mertcanfirat**

### CODE REVIEW

File: locales/tr_TR/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -1563,11 +1563,11 @@ msgstr "İşte beğeneceğinizi düşündüğümüz bazı şeyler"
 #. shown after the DuckDuckGo extension is installed
 msgctxt "welcome message"
 msgid "Here's what it does:"
-msgstr ""
+msgstr "İşte ne var:"
 
 msgctxt "new tab page"
 msgid "Hide Most Visited Sites"
-msgstr ""
+msgstr "En Çok Ziyaret Edilen Siteleri Gizle ""
```
Comment (jbarrett) [inline locales/tr_TR/LC_MESSAGES/duckduckgo.po]: Tests are failing because of the double quote at the end of this sting

Comment (mertcanfirat) [reply locales/tr_TR/LC_MESSAGES/duckduckgo.po]: I made the necessary corrections.

Comment (jbarrett) [reply locales/tr_TR/LC_MESSAGES/duckduckgo.po]: Not seeing the change. Do you need to `git push`?

Thanks!

**Comment by jbarrett** [comment]
Thanks @mertcanfirat!

**Pull request merged**